specifications/M1/auth.flow.puml

```diff
@@ -0,0 +1,19 @@
+@startuml
+!pragma layout smetana
+
+!include diagram.styles.puml
+
+autonumber
+
+participant "Client" as C
```

Comment:
The verifier is the system that performs cryptographic verification. For example, it may be a connector that provides data. It verifies the VP that was received from the client's Credential Service using cryptographic material the client provides (e.g. through a DID document)